esf_main.tex

```diff
@@ -161,18 +178,23 @@ \subsubsection{Description/concept}
     \end{table}
 
 \subsubsection{Wiring / additional circuitry}
-%shutdown schematic
-TODO
+	Figure \ref{shutdown_circuit} shows the wiring of the shutdown circuit.
+
+	\begin{sidewaysfigure}[p]
+        \includegraphics[width= 1\textheight]{shutdown_circuit}
+        \caption{Block Diagram of Shutdown Circuit}
```

Comment:
We're going to have to figure out how to make a numbered list of figures somewhere down the line. I'll look at last year's ESF for reference